plugin/gf/user.vim

```diff
@@ -35,7 +35,7 @@ function! s:define_ui_key_mappings()
   \              '<C-w>gf', '<C-w>gF']
     for mode_char in ['n', 'x']
       execute printf(
-      \   '%snoremap <Plug>(%s-%s) :<C-u>call gf#user#do("%s", "%s")<CR>',
+      \   '%snoremap <Plug>(%s-%s) <Cmd>call gf#user#do("%s", "%s")<CR>',
```

Comment:
Hi kana, the <Cmd> is introduced by `patch-8.2.1978` and the fix will bump requirement version.